app/controllers/photos_controller.rb

```diff
@@ -76,4 +81,11 @@ def set_photo
   def photo_params
     params.require(:photo).permit(:image, :comments_count, :likes_count, :caption, :owner_id)
   end
+
+  def ensure_user_is_authorized 
```

Comment:
this is related to the before action calling this, which is now no longer necessary thanks to pundit authorize! so can remove entirely or commented out is my suggestion